CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ source .venv/bin/activate
 ```
 Install Requirements
 ```sh
-pip install dev.requirements.txt
+pip install -r dev.requirements.txt
 ```
 Create a `secrets.py` file in the `pyconafrica` folder
```

Comment:
I do have some suggestions here;
1. Could we please outline any default environment variables that should be added to the `secrets.py`. if none, we could as  well mention that.
2. Could we please he explicit on exact location of the `secrets.py`. The root directory is called `pyconafrica`, as well the default app. For starters one could create the `secrets.py` in the root directory yet the intended location could be in default app.